### ISSUE TITLE
Quests Module not fully up to date.  Refresh?

### ISSUE THREAD

**Issue by noahgithub** [IrcDirk/Carbonite-Classic#326]
When Questing the add on is not fully up to date on all quests.  Some have missing waypoints and some are not in the database at all. Is it possible to give the quests a refresh?  Have seen this in both Hardcore and SoD. It has been working fantastic with the new update.

**Comment by IrcDirk** [comment]
Give me some examples.

**Comment by noahgithub** [comment]
Will have to gather specifics, but in SoD they have added new quests and activities that never happened In classic before. There are class quests to obtain runes that grant abilities for example.

**Comment by IrcDirk** [comment]
Yeah...give me those quest names or ids or something. It highly depends on that if they are in wowhead, then maybe i will be able to update it.

**Comment by noahgithub** [comment]
Here is an example plague talking about this for Druids. 

https://www.wowhead.com/classic/guide/season-of-discovery/druid-class-runes-discovery-location

**Comment by IrcDirk** [comment]
I dont see any quest on this site...

**Comment by noahgithub** [comment]
Will gather more specifics, that was one example of information not existing at all. Since those are not quests per se it might not be possible to add those.

**Comment by noahgithub** [comment]
From The Wreckage quest locator is off by a lot as well.  I could go on as there are a lot of them this way but I wonder if there is a way to do a refresh of the quests.  I have not run across a missing one yet, but I know there are a lot of them, will post when I find it.

**Comment by noahgithub** [comment]
The Disruption Ends, it asks you to get Kreenig Sarlsnout's Tusk. The objective is not in the database.

**Comment by IrcDirk** [comment]
Yeah it happens sometimes that wowhead has no map for one objective so there is no option to get this from there.

**Comment by noahgithub** [comment]
Is there code that I can run and present the results to you?

**Comment by IrcDirk** [comment]
What do u mean?

**Comment by noahgithub** [comment]
To scrape the site and prepare it for the add on.  Is there code that does the work or is it manual?

**Comment by noahgithub** [comment]
The Power To Destroy quest not in the database

**Comment by noahgithub** [comment]
@IrcDirk have you had any time to attempt a refresh and see if there are any quest updates?

**Comment by SyberHexen** [comment]
The one quest I've run across which needs a refresh is ["Warsong Supplies"](https://www.wowhead.com/classic/quest=6571/warsong-supplies)
1) Objective ['Logging Rope'](https://www.wowhead.com/classic/item=16743/logging-rope) is missing in DB.
2) Objective ['Warsong Axe Shipment'](https://www.wowhead.com/classic/object=178204/warsong-axe-shipment) location reports in wrong location.

**Comment by artscout** [comment]
I'm working on multiple quests data updates but this is slow enough progress. WIP
